plugin/goto_diagnostic.py

```diff
@@ -67,6 +69,7 @@ def input(self, args: dict) -> Optional[sublime_plugin.CommandInputHandler]:
         if uri == "$view_uri":
             try:
                 uri = uri_from_view(view)
+                return DiagnosticUriInputHandler(self.window, view, uri)
```

Comment:
Can we move this below the try/except block? I guess we don't expect it to throw so we shouldn't handle that gracefully.